sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -775,6 +795,14 @@ namespace Azure { namespace Data { namespace Tables {
        */
       std::vector<Models::TableEntity> TableEntities;
 
+      /**
+       * @brief Construct a new Query Entities Paged Response object.
+       *
+       * @param tableClient Table client.
+       */
+      QueryEntitiesPagedResponse(std::shared_ptr<TableClient> tableClient)
```

Comment:
Same comment, if we should make this private, if users are not supposed to create instances of these.